app/forms/flood_risk_engine/steps/correspondence_contact_telephone_form.rb

```diff
@@ -0,0 +1,38 @@
+
+
+module FloodRiskEngine
+  module Steps
+    class CorrespondenceContactTelephoneForm < BaseForm
+
+      def self.factory(enrollment)
+        enrollment.correspondence_contact ||= FloodRiskEngine::Contact.new(contact_type: :correspondence)
+        new(enrollment.correspondence_contact, enrollment)
+      end
+
+      def self.params_key
+        :correspondence_contact_telephone
+      end
+
+      property :telephone_number
+
+      # phone validation from https://github.com/daddyz/phonelib
+      validates(
+        :telephone_number,
+        phone: {
+          message: I18n.t("#{locale_key}.errors.telephone_number.invalid"),
+          allow_blank: true
+        },
```

Comment:
each to their own, but for me this format makes it hard to differentiate the different validators